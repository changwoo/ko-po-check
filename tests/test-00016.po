# -*-text-*-
# language/consistency
#
msgid ""
msgstr ""
"Project-Id-Version: test 15\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-03-06 00:16+0900\n"
"PO-Revision-Date: 2010-03-06 00:16+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 0);"

msgid ""
msgstr ""
"Project-Id-Version: test 15\n"
"POT-Creation-Date: 2010-03-06 00:16+0900\n"
"PO-Revision-Date: 2010-03-06 00:16+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;"

msgid "Field:"
msgstr "필드"

msgid "Open As..."
msgstr "...로 열기"

msgid "There should be a dot."
msgstr "마침표를 써야 합니다"

