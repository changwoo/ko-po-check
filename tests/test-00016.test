#!/bin/sh

FILTERS="language/consistency"

DIR="`dirname $0`"
NAME="`basename $0 .test`"
BUILDDIR=`pwd`
POFILE=$NAME.po
OUTFILE=$NAME.out

TFM=`mktemp`
export PYTHONPATH=..
python $BUILDDIR/../scripts/ko-po-check --filters="${FILTERS}" $POFILE > $TFM
diff -u $TFM $OUTFILE
RESULT=$?
rm -f $TFM
exit $RESULT
