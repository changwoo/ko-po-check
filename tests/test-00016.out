test-00016.po:29: 번역문이 원문과 같이 :으로 끝나야 합니다
test-00016.po:32: 번역문이 원문과 같이 ...으로 끝나야 합니다
test-00016.po:35: 번역문이 원문과 같이 .으로 끝나야 합니다
